config/application.rb

```diff
@@ -121,7 +121,7 @@ def preview?
 
     # @return [Boolean]
     def debug?
-      Rails.env.development? || review?
+      Rails.env.development?
```

Comment:
Is this getting reverted before merging?

Reply:
I don't think we need debug in review apps any more - a later branch I do add and env variable, but I'd like to avoid adding more env variables.  This is the point where I wish we had Redis so we could have a key value store to use for feature flags/functional switches....

Reply:
I'm down with it, a dev can always turn it back on if the review app needs it